src/rustc_lexer/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+authors = ["The Rust Project Developers"]
+name = "rustc_lexer"
+version = "0.1.0"
+edition = "2018"
+
+# [workspace]
```

Comment:
I think this should be removed before merging (I assume it's used for local testing?)

Reply:
Yeah, that’s for local testing and will be removed. In general, this is still in a very draft state: the final code will hopefully be much cleaner, but I do appreciate intermediary reviews!